show/muxcable.py

```diff
@@ -820,3 +820,196 @@ def switchmode(port):
         click.echo(tabulate(body, headers=headers))
         if rc == False:
             sys.exit(EXIT_FAIL)
+
+
+def create_result_dict(physical_port, mux_info_dict, result_dict):
```

Comment:
@jleveque should we move all this to y_cable.py or let it remain here ? 
this is in regards to the newer resultant dict you suggested. 
For example ? 
{
    "version_nic_active": "0.6MS",
    "version_nic_inactive": "0.5MS",
    "version_nic_next": "0.6MS",
    "version_self_active": "0.5MS",
    "version_self_inactive": "0.6MS",
    "version_self_next": "0.6MS",
    "version_peer_active": "0.6MS",
    "version_peer_inactive": "0.6MS",
    "version_peer_next": "0.6MS",
}

Reply:
refactored, new API's used here